packages/wallet/backend/src/account/service.ts

```diff
@@ -31,10 +35,19 @@ interface IAccountService {
   withdrawFunds: (args: WithdrawFundsArgs) => Promise<void>
 }
 
+type CreateExchangeQuote = {
+  userId: string
+  accountId: string
+  assetCode: string
+  amount: number
+}
+
 interface CountriesServiceDependencies {
   rapyd: RapydClient
   rafiki: RafikiClient
   logger: Logger
+  quoteServiceGetter: () => QuoteService
```

Comment:
keep it just as quoteService, we don't use this naming for dependencies anywhere else